seedpod_ground_risk/ui_resources/aircraft_options.py

```diff
@@ -12,7 +12,6 @@ def aircraft_list():
 def add_aircraft(new_ac):
     ac_list = AIRCRAFT_LIST
     ac_list[f"{new_ac['name']}"] = new_ac
-    if 'name' in ac_list[f"{new_ac['name']}"]: del ac_list[f"{new_ac['name']}"]['name']
```

Comment:
This doesn't look like it deals with name collisions for new aircraft. Since the name is essentially being used as the UID, when it comes around to lookup, it should be unique.